install.sh

```diff
@@ -195,12 +195,25 @@ main() {
   tput setaf 1;
 
   isNonInteractive=false
-  while getopts nh opt; do
-     case $opt in
-        n) isNonInteractive=true ;;
-        h) usageFunction ;;
-        ?) usageFunction ;;
-     esac
+  # Get args from sudo or directly
+  args="${@:-${SUDO_COMMAND#*/install.sh }}"
```

Comment:
Not exactly understandable/readable, because `SUDO_COMMAND` is defined nowhere, so it'll be set anyway.

Reply:
https://askubuntu.com/a/953497

SUDO_COMMAND is an environment variable set by sudo only in the environment of the process started by it (and inherited by any child processes).